docs/product-user-guide/src/main/asciidoc/proc-business-central-settings-artifacts.adoc

```diff
@@ -0,0 +1,4 @@
+[[business_central_settings_artifacts]]
+= Artifacts
+
+In the Artifacts perspective you can manage the content of the Maven repository. The artifacts that you built with your projects are listed here, along with their GNU artifact version (GAV). You can view the contents an artifact, as well as upload and download artifacts.
```

Comment:
The (maven) artifacts (usually jar files, in the context of our project named "Kjars" - K for knowledte) are result of building projects. I'd change it to "artifacts that you build FROM your projects".

"along with their GNU artifact version" -> Along with their GroupId/AftifactId/Version. 
Not sure how to present this in written form - see [this page](https://maven.apache.org/guides/mini/guide-naming-conventions.html) to understand what GAV stands for.